[ADD] C6: communication information for contacts

remytms opened the pull request:
Add modules to complete the C6 task for Pro Vélo.

This PR contains : 

- `partner_communicate_tag`
- `partner_communicate_lang`
- `partner_communicate_press`

remytms commented:
I have created a new branch in the Houssine repos. If you prefer that I work on my on copy of the repos, just tell me. :)

robinkeunen commented:
parfait :-)

On Jun 28 2018, at 8:57 am, Rémy Taymans <notifications@github.com> wrote:
>
> I have created a new branch in the Houssine repos. If you prefer that I work on my on copy of the repos, just tell me. :)
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub (https://github.com/houssine78/addons/pull/21#issuecomment-400932404), or mute the thread (https://github.com/notifications/unsubscribe-auth/AC8Hh6ruoPhLXGOIIahHLg4G54WCxCQZks5uBH5agaJpZM4U61yL).
>
>

remytms committed:
[ADD] Module partner_communication_tag

This module let you add tags to a contact this tags are aimed to be used
for communication purposes.

remytms committed:
[ADD] Module partner_communication_lang

Let add a communication language to partners.

remytms committed:
[ADD] Module partner_communication_press

Add type of press to send to contacts.

remytms commented:
Une relecture est la bienvenue.

remytms commented:
Je peux aussi diviser cette PR en trois partie si c'est plus facile pour vous. :)

houssine78 commented on partner_communication_lang/__openerp__.py:
```diff
@@ -0,0 +1,31 @@
+    'author': 'Rémy Taymans',
+    'license': 'AGPL-3',
+    'version': '9.0.1.0',
+    'website': "https://github.com/houssine78/addons",
```
www.coopiteasy.be

houssine78 commented on partner_communication_lang/views/partner_communication_lang.xml:
```diff
@@ -0,0 +1,51 @@
+     Copyright 2018 Rémy Taymans <remytaymans@gmail.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<openerp>
```
use <odoo> tag
